packages/zowe-explorer/CHANGELOG.md

```diff
@@ -19,7 +21,6 @@ All notable changes to the "vscode-extension-for-zowe" extension will be documen
 
 ### New features and enhancements
 
-- Multiple select for hide session [#1555](https://github.com/zowe/vscode-extension-for-zowe/issues/1555)
```

Comment:
Not sure if this was intentional, but I noticed this line was removed from the `2.5.0` changelog.

Reply:
yeah, it seems like it was a duplicate introduced in one of the merges from main 😋

Reply:
Ohh I see, I overlooked that - thanks for the clarification